platform/osx/os_osx.mm

```diff
@@ -1516,7 +1522,7 @@ static void keyboardLayoutChanged(CFNotificationCenterRef center, void *observer
 }
 
 void OS_OSX::push_input(const Ref<InputEvent> &p_event) {
-
+	//toger why is ther another function alternative would be to just use input->input->parse_input_event(ev);
```

Comment:
It's a public method so I thought it might be used by other parts of the code base to push input events directly, but it doesn't seem to be the case: https://github.com/godotengine/godot/search?utf8=%E2%9C%93&q=push_input&type=

What you suggest is what is done for X11, so I guess it could be used for OSX too: https://github.com/godotengine/godot/blob/5b3709d3096df737b8bb2344446be818b0389bfe/platform/x11/os_x11.cpp#L1361